x-pack/plugin/security/src/internalClusterTest/java/org/elasticsearch/xpack/security/transport/filter/IpFilteringIntegrationTests.java

```diff
@@ -57,6 +59,16 @@ protected Settings nodeSettings(int nodeOrdinal, Settings otherSettings) {
             .build();
     }
 
+    @Before
+    public void waitForSecurityIndex() throws Exception {
+        assertSecurityIndexActive();
```

Comment:
Waiting for active `.security` index before test and deleting it below in order to avoid tripping assertions: java.lang.AssertionError: FixedBitSet cache size must be 0 on node: